api/nakadi-event-bus-api.yaml

```diff
@@ -1177,8 +1177,7 @@ definitions:
         allOf:
           - $ref: '#/definitions/EventTypeSchema'
         description: |
-          The schema for this EventType. This is expected to be a json schema in yaml format (other formats
-          might be added in the future).
```

Comment:
Can you please point me this place, because for me it is a string https://github.com/zalando/nakadi/blob/aruha-184-fix-schema-doc/api/nakadi-event-bus-api.yaml#L1223